### PR TITLE
Add GraqhQL search documentation

### PR DESCRIPTION
**Pull request by bjornandre**

### CODE REVIEW

File: docs/administration.adoc [hunk 1]
```diff
@@ -37,6 +37,14 @@ The persistence providers share a common configuration located at `linked-data-s
 |===
 |Key |Default value |Description
 
+|graphql.enabled
+|false
+|Enables the GraphQL API
+
+|graphql.search.enabled
```
Comment (hadrienk) [inline docs/administration.adoc]: Where do I set the Solr URL?

Comment (hadrienk) [reply docs/administration.adoc]: That's what `search.index.provider=` right?

Comment (bjornandre) [reply docs/administration.adoc]: Well, no. The search.index.provider is used with Java SPI to configure which search provider class to instantiate. The solr url is specific to the solr implementation (which is a separate project)

Comment (bjornandre) [reply docs/administration.adoc]: Fixed

**Pull request merged**